slicedimage/io.py

```diff
@@ -125,7 +125,7 @@ def default_tile_writer(tile, fh):
         tile.write(fh)
         return ImageFormat.NUMPY
 
-    def generate_toc(self, imagestack, path, *args, **kwargs):
+    def generate_toc(self, imagestack, path, pretty=False, *args, **kwargs):
```

Comment:
generate_contents? i'm fine w/ pretty=False

Reply:
out of the scope of this PR.